examples/distributed-tracing/src/composer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "open-telemetry/distributed-tracing-example",
+  "type": "application",
+  "homepage": "https://opentelemetry.io/docs/php",
+  "license": "Apache-2.0",
+  "require": {
+    "php": ">=8",
+    "open-telemetry/opentelemetry": "dev-main",
+    "guzzlehttp/guzzle": "*",
+    "php-http/guzzle7-adapter": "*",
+    "slim/slim": "~4",
+    "php-di/php-di": "^6.3",
+    "php-di/slim-bridge": "^3.2"
+  }
+}
```

Comment:
Could you please commit your composer.lock, to make sure this example always works?

Reply:
I can. I'd better also change "composer update" to "composer install" if we want it to always work...

Reply:
and, to be able to commit a lock file I'm waiting on #484 - dev-main in packagist does not have some recent changes that this example uses.

Reply:
I see. 
I never noticed that composer is using packagist to resolve the latest commit for branches.

Reply:
I can change the composer config so that it pulls the library from github rather than packagist? The only reason I didn't do it that way was because I saw that packagist is the official distribution method (and avoids potentially having to deal with github tokens to install)

Reply:
Yes, I guess that's a good temporary workaround until auto-update is enabled and/or packagist catches up.